data/channels/dirksocialist.yml

```diff
@@ -0,0 +1,29 @@
+description: Greetings! I am Comrade Dirk and you are welcome to my channel 'Dirk
+  Socialist' It may be no surprise that the channel Dirk Socialist is ran by a socialist.
+description_more: Although to many the term 'Socialism' appears to be vulgar, I believe
+  that most people would be interested in social ownership of the means of production
+  if the idea were not so stigmatized. <br/><br/> As such, this channel is intended
```

Comment:
How do these <br/><br/> tags get processed?

Reply:
It gets included on the channel specific page: https://github.com/breadtubetv/breadtubetv/blob/2ef13e2874c7191989d2ecdbfa419d32b11d6d6c/layouts/channels/single.html#L15